pootle/static/js/shared/components/TimeSince.js

```diff
@@ -75,8 +75,7 @@ const TimeSince = React.createClass({
     return (
       <time
         className="extra-item-meta"
-        title={this.props.title}
-        dateTime={this.props.dateTime}
+        {...this.props}
```

Comment:
The commit message says _Make it possible to overridde TimeSince props_. Note that it's not necessary to remove `title` and `dateTime` for that purpose and in fact I suggest you keep them for explicitness.